src/main.rs

```diff
@@ -0,0 +1,96 @@
+use std::ffi::CString;
```

Comment:
This file is just for fun, for something to work if run with "cargo run". All the code is used in the tests. Here I just added few simple examples.